src/casp/services/deploy/Dockerfile.pytorch

```diff
@@ -1,11 +1,19 @@
 FROM bitnami/pytorch:latest
-
 WORKDIR /app
+USER root
 
+# Copy Files and Secrets
 COPY requirements.txt .
+COPY src .
```

Comment:
This is also changed because of problem with parsing env secret.

Reply:
the problem with moving this earlier is that every chance to a source file will cause the rest of the steps to have to be re-run.  At the very least, I would move the `pip install` step up above the `COPY src .` step since that's expensive/slow and doesn't change as often as the source code